.vscode/settings.json

```diff
@@ -1,136 +1,118 @@
-        "cstddef": "cpp",
-        "cstdint": "cpp",
-        "cstdio": "cpp",
-        "cstdlib": "cpp",
-        "cstring": "cpp",
-        "ctime": "cpp",
-        "cwchar": "cpp",
-        "cwctype": "cpp",
-        "deque": "cpp",
-        "exception": "cpp",
-        "fstream": "cpp",
-        "future": "cpp",
-        "initializer_list": "cpp",
-        "iomanip": "cpp",
-        "ios": "cpp",
-        "iosfwd": "cpp",
-        "iostream": "cpp",
-        "istream": "cpp",
-        "limits": "cpp",
-        "list": "cpp",
-        "locale": "cpp",
-        "map": "cpp",
-        "memory": "cpp",
-        "mutex": "cpp",
-        "new": "cpp",
-        "optional": "cpp",
-        "ostream": "cpp",
-        "queue": "cpp",
-        "ratio": "cpp",
-        "set": "cpp",
-        "sstream": "cpp",
-        "stack": "cpp",
-        "stdexcept": "cpp",
-        "streambuf": "cpp",
-        "string": "cpp",
-        "string_view": "cpp",
-        "system_error": "cpp",
-        "thread": "cpp",
-        "tuple": "cpp",
-        "type_traits": "cpp",
-        "typeinfo": "cpp",
-        "unordered_map": "cpp",
-        "variant": "cpp",
-        "vector": "cpp",
-        "__nullptr": "cpp",
-        "__string": "cpp",
-        "chrono": "cpp",
-        "compare": "cpp",
-        "concepts": "cpp",
-        "numeric": "cpp",
-        "random": "cpp",
-        "algorithm": "cpp",
-        "filesystem": "cpp",
-        "cfenv": "cpp",
-        "csignal": "cpp",
-        "strstream": "cpp",
-        "typeindex": "cpp",
-        "hash_map": "cpp",
-        "any": "cpp",
-        "barrier": "cpp",
-        "charconv": "cpp",
-        "cinttypes": "cpp",
-        "codecvt": "cpp",
-        "coroutine": "cpp",
-        "csetjmp": "cpp",
-        "execution": "cpp",
-        "format": "cpp",
-        "forward_list": "cpp",
-        "functional": "cpp",
-        "iterator": "cpp",
-        "latch": "cpp",
-        "memory_resource": "cpp",
-        "numbers": "cpp",
-        "ranges": "cpp",
-        "regex": "cpp",
-        "scoped_allocator": "cpp",
-        "semaphore": "cpp",
-        "shared_mutex": "cpp",
-        "source_location": "cpp",
-        "span": "cpp",
-        "stop_token": "cpp",
-        "syncstream": "cpp",
-        "unordered_set": "cpp",
-        "utility": "cpp",
-        "valarray": "cpp",
-        "xfacet": "cpp",
-        "xhash": "cpp",
-        "xiosbase": "cpp",
-        "xlocale": "cpp",
-        "xlocbuf": "cpp",
-        "xlocinfo": "cpp",
-        "xlocmes": "cpp",
-        "xlocmon": "cpp",
-        "xlocnum": "cpp",
-        "xloctime": "cpp",
-        "xmemory": "cpp",
-        "xstddef": "cpp",
-        "xstring": "cpp",
-        "xtr1common": "cpp",
-        "xtree": "cpp",
-        "xutility": "cpp",
-        "*.ipp": "cpp",
-        "*.tcc": "cpp",
-        "rope": "cpp",
-        "slist": "cpp",
-        "print": "cpp"
-    },
-    "C_Cpp.default.compilerPath": "/usr/bin/clang++"
-}
+{
+    "files.associations": {
```

Comment:
I don't know how to analyze this change.  Surely it's not part of ”MSVC C++20 Support“ though.

Reply:
This file really shouldn't be here - I'll do the two step delete and add to .git-ignore.